app/views/devise/passwords/edit.html.erb

```diff
@@ -1,9 +1,15 @@
-<h2>Change your password</h2>
+<% content_for :title do "Change your password" end %>
+
+<% content_for :alerts do render "application/alerts" end %>
```

Comment:
Single line blocks should use `{ }`. So `content_for :alerts { render "application/alerts" }`.

Same for above.